### PR TITLE
[#841] Alphabetized resource type list when creating a resource.

### PR DESCRIPTION
**Pull request by mauriel133**

### CODE REVIEW

**Comment by hydrobuild** [comment]
Can one of the admins verify this patch?

**Comment by mohamedmorsy** [comment]
@mauriel133 could you please include swat model instance resource after the model instance resource? As this swat model instance resouce is specific model instance resource. @dtarb what do you think about that?

**Comment by mauriel133** [comment]
@mohamedmorsy,  how about something like this:
capture

**Comment by mohamedmorsy** [comment]
@mauriel133 that would be great. Could you change the title to Model resources and do the following arrangement: Model Program, Model Instance, and SWAT Model Instance. Or just get the Model Program out of the list.

**Comment by mauriel133** [comment]
@mohamedmorsy, I moved it out of the group. @dtarb, @horsburgh What do you think about this groups approach?

**Comment by mauriel133** [comment]
Done.

**Comment by horsburgh** [comment]
@mauriel133 - the groups seem fine to me.

**Comment by dtarb** [comment]
I think the group approach is a good idea.  I would prefer the model related group header to be "Modeling".  I think that Script may also belong in Modeling.

**Comment by mauriel133** [comment]
Applied the suggestions above.

File: hs_core/templates/pages/create-resource.html [hunk 1]
```diff
@@ -45,21 +45,31 @@
...
-                <option value="SWATModelInstanceResource">SWAT Model Instance</option>
                 <option value="NetcdfResource">Multidimensional (NetCDF)</option>
-                <option value="TimeSeriesResource">Time Series</option>
-                <option value="RefTimeSeriesResource">HIS Referenced Time Series</option>
                 <option value="ToolResource">Web App</option>
-                <option value="ScriptResource">Script</option>
-{#                <option value="InstResource">RHESSys Instance Resource</option> #}
-                <option value="GeographicFeatureResource">Geographic Feature (ESRI Shapefiles)</option>
 
+                <optgroup label="Geographic Resources">
```
Comment (pkdash) [inline hs_core/templates/pages/create-resource.html]: I assume this grouping rather than ordering based on resource type name.

**Comment by pkdash** [comment]
+1

**Pull request merged**